RileyLinkKit/PumpOps.swift

```diff
@@ -55,6 +55,23 @@ public class PumpOps {
         }
     }
     
+    public func readSettings(_ completion: @escaping (Either<ReadSettingsCarelinkMessageBody, Error>) -> Void)  {
+        device.runSession(withName: "Read pump settings") { (session) -> Void in
+            let ops = PumpOpsSynchronous(pumpState: self.pumpState, session: session)
+            do {
+                let response: ReadSettingsCarelinkMessageBody = try ops.messageBody(to: .readSettings)
+                DispatchQueue.main.async { () -> Void in
```

Comment:
I meant the main queue dispatch, since the vast majority of callers are invoking PumpOps from a background queue anyway.

There's not currently a consistent behavior in these commands, by my count: 4.5 commands do dispatch to main and the other 8 don't.